README.md

```diff
@@ -149,66 +149,75 @@ When you leave a critical section, disable LOG again, like this:
 Sentry Integration
 ------------------
 
-This logger is easily integrated with Sentry.  For Sentry integration, set the LOG variable 'sentryCallback'
-to point to your Sentry implementation.  The example below shows a possible implementation.
-
-	class SentryHandler() {
-
-		companion object {
-
-			fun initializeSentry(clientName: String, environment: String, dsn: String) {
-				LOG.m()
-				LOG.sentryCallback = Companion::reportToSentry
-				val options = SentryOptions()
-				options.dsn = dsn
-				options.isAttachStacktrace = true
-				options.enableUncaughtExceptionHandler = true
-				options.setDebug(BuildConfig.DEBUG)
-				options.environment = environment
-				options.sentryClientName = clientName
-				options.serverName = clientName
-				Sentry.init(options)
-			}
-
-			fun reportToSentry(t: Throwable?, tag: String, message: String) {
-				LOG.m()
-				if (Sentry.isEnabled()) {
-					Sentry.setTag("info", tag)
-					Sentry.setTag("message", message)
-					val sentryMessage = Message()
-					sentryMessage.message = message
-					val event = SentryEvent()
-					if (t != null) {
-						val sentryException = SentryException()
-						sentryException.threadId = Thread.currentThread().id
-						sentryException.value = t.message
-						//sentryException.stacktrace = getSentryStackTrace()
-						event.exceptions = listOf(sentryException)
-					}
-					event.level = when (tag) {
-						LOG.WTF -> SentryLevel.FATAL
-						LOG.ERROR -> SentryLevel.ERROR
-						LOG.WARN -> SentryLevel.WARNING
-						LOG.DEBUG -> SentryLevel.DEBUG
-						else -> SentryLevel.INFO
-					}
-					event.message = sentryMessage
-					addBreadcrumbs(event)
-					Sentry.captureEvent(event)
-				}
-			}
-
-			// Capture the last 50 lines of logcat to use as Breadcrumbs
-			private fun addBreadcrumbs(event: SentryEvent) {
-				LOG.m()
-				val process = Runtime.getRuntime().exec("logcat -t 50 --pid ${android.os.Process.myPid()} *:D")
-				val lines = process.inputStream.bufferedReader().use { it.readText() }.split("\n")
-				for (line in lines) {
-					event.addBreadcrumb(line)
-				}
-			}
-		}
-	}
+This logger is easily integrated with [Sentry](https://sentry.io).  For Sentry integration, set the LOG variable 'theSentryCallback'
+to point to your Sentry implementation.  The example below shows a possible implementation.  You may also want to add a default exception handler.
+Note that only ERROR and WTF messages are currently sent to Sentry.
+
+    class SentryHandler() {
```

Comment:
If you wrap this whole class in three backticks, you can also define the language which will improve the code styling in GitHub. Example which spaces to prevent formatting (I added a forward slash so it does not render):

\```Kotlin
if (isAwesome) {
    return true
}
\```

Reply:
nice!  i like that.